scripts/run-crawlers.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Utility script for running crawlers
+
+re='^[0-9]+$'
+if ! [[ $1 =~ $re ]] ; then
```

Comment:
Hm, this works if we just want type but I set up my old version hoping to accept parameters like which crawler to run in the future - doubt we'll really need it though